src/lib/clients/IOClients/apps/Builder.ts

```diff
@@ -111,9 +114,11 @@ export class Builder extends AppClient {
   }
 
   public relinkApp = (app: string, changes: ChangeToSend[], params: RequestParams = {}) => {
+    console.info(`linkID: ${this.linkID}`)
```

Comment:
The client shouldn't be responsible for logging

Reply:
I think the caller should be responsible

Reply:
Also, when logging on toolbelt, use the `logger` object - there's a `logger.ts` that exports a logger